docs/install/configuration.md

```diff
@@ -138,6 +138,27 @@ Option        | Description
 `telemetry.frontend.posthog.capture_pageview` | FlowForge is designed as to provide custom posthog `$pageview` events that provide more detail on navigation than the default, and suit a single page application better. As such, we recommend setting this to false in order to prevent duplicate `pageleave`/`pageview` events firing. Default: `true`
 
 
+## Rate Limiting configuration
+
+By default, rate limiting is disabled and the platform will not rate limit any requests.
+
+To enable rate limiting, you can set the `rate_limits.enabled` option to `true`. 
+
+When enabled, all routes will be limited to 1000 requests per 1 minute window. These defaults can be adjusted by setting values in the configuration options listed below.
+
+NOTE: Some routes are prohibited from being rate limited, such as the logout and acl routes.
```

Comment:
While I understand why this is documented I wonder if it just makes it a target for attack

Reply:
Good question Ben.  Open to suggestions.